usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/TestFrameworkAssertions.java

```diff
@@ -39,122 +42,216 @@
  * @author m4rkmckenna on 11/11/2015.
  */
 public class TestFrameworkAssertions {
-    private static final Logger LOG = LoggerFactory.getLogger(TestFrameworkAssertions.class);
+
+    public static final String IS_NULL = "isNull";
+    public static final String NOT_NULL = "notNull";
+    public static final String IS_EQUAL_TO = "isEqualTo";
```

Comment:
I have a personal preference of not supporting lots of variants. It feels nice sometimes, but then when someone starts looking at different examples and sees different text in each, it looks confusing (*). But no strong feelings.

(*) there are exceptions to that, such as in Duration where we accept `1h` and `1hr` etc.